mappings/net/minecraft/command/argument/BlockPosArgumentType.mapping

```diff
@@ -6,10 +6,17 @@ CLASS net/minecraft/class_2262 net/minecraft/command/argument/BlockPosArgumentTy
 	METHOD listSuggestions (Lcom/mojang/brigadier/context/CommandContext;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 context
 		ARG 2 builder
+	METHOD method_48298 getLoadedBlockPos (Lcom/mojang/brigadier/context/CommandContext;Lnet/minecraft/class_3218;Ljava/lang/String;)Lnet/minecraft/class_2338;
+		ARG 0 context
+		ARG 1 world
+		ARG 2 name
+	METHOD method_48299 getBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/class_2338;
+		ARG 0 context
+		ARG 1 name
 	METHOD method_9696 getLoadedBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/class_2338;
 		ARG 0 context
 		ARG 1 name
-	METHOD method_9697 getBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/class_2338;
+	METHOD method_9697 getInBoundsBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/class_2338;
```

Comment:
The bounding box check here is not that useful, since it's not height limit aware. Perhaps just `getValidBlockPos`?

Reply:
I guess that would work